lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -987,8 +987,14 @@ tearDown ctx = do
         let endpoint = "v2/wallets" </> wal ^. walletId
         d <- request @Value ctx ("DELETE", endpoint) None Empty
         expectResponseCode HTTP.status204 d
+    respByron <-
```

Comment:
we don't want to have separate `tearByronDown`? And check if tearing down byron wallet do not affect wallet? Etc?

Reply:
@paweljakubas wrote:
>  we don't want to have separate tearByronDown? And check if tearing down byron wallet do not affect wallet? Etc?
@piotr-iohk What's your opinion on this?

Reply:
such tests (i.e. trying to delete byron with new endpoint and vice versa) I was planning to add in the ByronWallet.hs suite actually. They are _planned_ in the spreadsheet for now:
https://docs.google.com/spreadsheets/d/1fadIA_j4nCd3FbylPo5J8K9Fy6tixSC2T9V3xHKuUvU/edit#gid=0

`BYRON_DELETE`

Reply:
ok, so `tearDown` should be ok after that change. and there is no need for separate `tearByronDown`